consensus/src/runway/mod.rs

```diff
@@ -914,23 +901,38 @@ pub(crate) async fn run<H, D, US, UL, MK, DP, FH, SH>(
     MK: MultiKeychain,
     SH: SpawnHandle,
 {
+    let RunwayIO {
+        data_provider,
+        finalization_handler,
+        backup_write,
+        backup_read,
+        _phantom: _,
```

Comment:
You can't use something like `..,` here?

Reply:
I like getting the additional check from the compiler that everything has been unpacked, since here actually everything should be. If there were more such fields I would use `..`, but in this case I would prefer not to.